libs/constrained-generators/src/Constrained/Test.hs

```diff
@@ -356,7 +375,7 @@ mapPairSpec = constrained' $ \m s ->
 
 mapEmptyDomainSpec :: Spec BaseFn (Map Int Int)
 mapEmptyDomainSpec = constrained $ \m ->
-  subset_ (dom_ m) mempty
+  subset_ (dom_ m) (Lit Set.empty)
```

Comment:
I strongly disagree, the goal here is to keep things close to Haskell syntax to avoid overhead when reading the code. Introducing extra `Lit` when it's not necessary I think makes the code more difficult to read.

Reply:
It now reads
```
subset_ (dom_ m) mempty  -- mempty in the Monoid instance (Term fn (Set a))
```